addons/website_sale/views/templates.xml

```diff
@@ -1092,8 +1235,12 @@
                             <i class="oi oi-close" role="presentation"/>
                         </a>
                     </div>
+                    <t
+                        t-set="visible_attributes"
+                        t-value="len([a for a in attributes if a.value_ids and len(a.value_ids) &gt; 1])"
+                    />
```

Comment:
I'm not quite sure that is working at the moment, I've tested it can I seems like the attributes are not collapsed when I open the page, can you quickly check this and tell me if I'm testing it wrongly ?

Reply:
Indeed you're right I tested with values 🤦‍♂️ Thanks for the heads up!